src/screens/Login.js

```diff
@@ -8,25 +8,30 @@ function Login({ navigation }) {
   const [callbacks, setCallbacks] = useState([]);
   const [nxt, setNxt] = useState(null);
```

Comment:
Not a huge fan of the naming here. Could you explain the difference between `nxt` and `next`?

Reply:
renamed.